AmplifyPlugins/Predictions/AWSPredictionsPluginIntegrationTests/IdentifyBasicIntegrationTests.swift

```diff
@@ -252,11 +252,11 @@ class IdentifyBasicIntegrationTests: AWSPredictionsPluginTestBase {
                 }
                 XCTAssertFalse(data.fullText.isEmpty)
                 XCTAssertFalse(data.words.isEmpty)
-                XCTAssertEqual(data.words.count, 28)
+                XCTAssertEqual(data.words.count, 33)
```

Comment:
Since this is all AI, I doubt whether this value can be used in test, it might change all different execution. I am keeping this for now and running in CI will give us more input on whether to remove this or not.